keyboards/sauce/mild/readme.md

```diff
@@ -0,0 +1,34 @@
+# Sauce Mild
+
+Mild is a basic TKL keyboard
+
+* Keyboard Maintainer: [Sauce](https://github.com/andyywz)
+* Hardware Supported: Mild pcb
+* Hardware Availability: -
+
+Make example for this keyboard (after setting up your build environment):
+
+    make sauce/mild:default
+
+Flashing example for this keyboard:
+
+    make sauce/mild:default:flash
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
+
+## Copyright
+
+Copyright 2021 Andy Yong (Sauce)
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
```

Comment:
```suggestion
```

Reply:
Oh @drashna requested that I add the copyright in this file, should I remove it?

Reply:
I'll just add this now and if its a problem I will revert the change :D